addon/TelemetryRappor.jsm

```diff
@@ -14,10 +14,29 @@ const PREF_RAPPOR_PATH = "toolkit.telemetry.rappor.";
 const PREF_RAPPOR_SECRET = PREF_RAPPOR_PATH + "secret";
 
 Cu.import("resource://gre/modules/Console.jsm");
+Cu.import("resource://gre/modules/Services.jsm");
+Cu.importGlobalProperties(['crypto']);
 
 const console = new ConsoleAPI({prefix: "shield-study-rappor"});
```

Comment:
According to the [documentation](https://developer.mozilla.org/en/docs/Mozilla/JavaScript_code_modules/Log.jsm), this module is still under development, and it's API is not well-defined, so it can change. I don't know if it's secure to use it on production.